kotlin-mbedtls-netty/src/main/kotlin/org/opencoap/ssl/netty/DatagramPacketWithContext.kt

```diff
@@ -25,7 +25,7 @@ import java.net.InetSocketAddress
 class DatagramPacketWithContext(
     data: ByteBuf,
     recipient: InetSocketAddress?,
-    sender: InetSocketAddress,
+    sender: InetSocketAddress?,
```

Comment:
Currently this class is used only for datagram receiving. `sender` must always be set. Now, when we want to also send this datagram, with setting recipient, but not setting the sender (because it will be set by underlying implementation).

Reply:
I see.. then it would be good to have test like that.